src/game/Tools/Formulas.h

```diff
@@ -151,13 +151,13 @@ namespace MaNGOS
             for (uint8 i = 1; i <= 13; i++)
             {
                 honor = 0.0f;
-                tempSt = sObjectMgr.GetHonorStandingByPosition(sc.BRK[i], team);
+                std::optional<HonorStanding> tempSt = sObjectMgr.GetHonorStandingByPosition(sc.BRK[i], team);
                 if (tempSt)
                 {
-                    honor += tempSt->honorPoints;
+                    honor += tempSt.value().honorPoints;
```

Comment:
pretty sure std::optional has a -> operator. Or was there a reason for not using it.

Reply:
Good catch. Using the -> operator is beter here. I amended the PR.

Reply:
Mhm, for the future, squash a small pr like this into a single commit. I will do it myself and merge it, this was my only remark.